sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -368,7 +368,7 @@ export class ManagementClient extends LinkEntity {
    * @param {SendRequestOptions} [options] Options that can be set while sending the request.
    * @returns {Promise<Date>} Promise<Date> New lock token expiry date and time in UTC format.
    */
-  async renewLock(lockToken: string, options?: SendRequestOptions): Promise<Date> {
+  async renewLock(lockToken: string, options?: SendRequestOptions, associatedLinkName?: string): Promise<Date> {
```

Comment:
Looks like `renewLock` and `renewSessionLock` are the only 2 operations in this class that are taking in the `SendRequestOptions`. But, going by the pattern in which any of the operations from this class are being used, no caller sends any of these options.

Can you please remove this `options` from the input arguments? It would make usage of this function much more cleaner i.e  compare
`renewLock(lockToken, undefined, receiverName` with `renewLock(lockTojen, receiverName`.

Reply:
It seems to be used at - https://github.com/Azure/azure-sdk-for-js/blob/8b35c88281de9bd4c3f5fcd3411f186fdbabebbf/sdk/servicebus/service-bus/src/session/messageSession.ts#L1144 correct?

Reply:
Ah, that's for renewSessionLock. Good point.

Now that it has been concluded that a lot of other management operations will need the associated link name, lets have the parameter for that be the second parameter and push options to the last.

Reply:
Also, lets rename this parameter to `associatedLinkName`.
Given that we intend to do this to some of the other operations as well, `associatedLinkName` is better suited for the purpose it gets used for on the service side

Reply:
Updated.